CONTRIBUTING.md

```diff
@@ -160,7 +163,16 @@ At the time of writing, the repository includes the following crates:
 
 ### Example: Adding a new lint rule
 
-At a high level, the steps involved in adding a new lint rule are as follows:
+To add a new rule, you can run the helper script `python scripts/add_rule.py` from within the ruff repository, which will take the following values:
```

Comment:
being able to run a script to reduce the amount of things needed for new users will make new contributions easier to make